### PR TITLE
Add overridable cacertfile

### PR DESCRIPTION
**Pull request by easink**
Add a way to override CAStore.

### CODE REVIEW

**Comment by josevalim** [comment]
Hi! Instead of an environment variable, can you please use the application environment, as the other configurations?  Thank you!

File: config/config.exs [hunk 1]
```diff
@@ -5,3 +5,5 @@ config :esbuild,
   another: [
     args: ["--version"]
   ]
+
+config :esbuild, cacerts_path: System.get_env("ESBUILD_CACERTS_PATH")
```
Comment (josevalim) [inline config/config.exs:9]: You will have to set this line in your project, if necessary. :) Other than that, it looks good!

```suggestion
```

**Comment by easink** [comment]
Yes, of course, and thank you, now fixed.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**